sklearn/manifold/tests/test_locally_linear.py

```diff
@@ -36,7 +36,7 @@ def test_barycenter_kneighbors_graph():
 def test_lle_simple_grid():
     # note: ARPACK is numerically unstable, so this test will fail for
     #       some random seeds.  We choose 2 because the tests pass.
-    rng = np.random.RandomState(2)
+    rng = np.random.RandomState(42)
```

Comment:
We should probably update the statement above regarding `2` :)

Reply:
Done, I rather test the behavior before.